juno-arb/ops/gcp/cloudrun/Readme.md

```diff
@@ -0,0 +1,42 @@
+Cloudrun is if you want the easiest, lowest overhead way of running a bot. If you want more flexibility and customizability, check out the `terraform` config.
+
+Since a job times out after a maximum of 1 hour, we get around that by scheduling this job to run every hour, at minute 0
```

Comment:
What are the pre-reqs? Looks like we need to gcloud cli installed? An account logged in as well? 

Let's list them explicitly